src/intrprtr.c

```diff
@@ -4682,7 +4682,22 @@ void            IntrInfoBegin( void )
 
 }
 
-Obj             InfoDecision;
+static Obj InfoDecision;
+static Obj InfoDecisionFast;
+static Obj IsInfoClassListRep;
+
+Obj InfoCheckLevel(Obj selectors, Obj level)
+{
+    if (IS_POS_INTOBJ(level) &&
+        CALL_1ARGS(IsInfoClassListRep, selectors) == True) {
+        Obj index = ELM_PLIST(selectors, 1);
+        return CALL_3ARGS(InfoDecisionFast, index, selectors, level);
```

Comment:
OK. But I am concerned that this might be forgotten. Can you record it somewhere? Either in a TODO comment right in the code (which is nice because one will stumble over it when digging through the `Info` code), or as an issue, or both?